### ISSUE TITLE
RGB Invoice Pay2Vout baid64 encode error

### ISSUE THREAD

**Issue by Crayon-Shin-chan-1024** [RGB-WG/rgb-std#212]
```
rgb -d .bob invoice $RGB20_CONTRACT RGB20Fixed 2000 --address-based
```

raises error

```
# thread 'main' panicked at /Users/cairo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/rgb-invoice-0.11.0-beta.6/src/parse.rs:258:14:
# address payload always less than 32 bytes: Error { kind: WriteZero, message: "failed to write whole buffer" }
# note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

This might be due to
https://github.com/RGB-WG/rgb-std/blob/master/invoice/src/parse.rs#L238-L261

the p2tr address scriptpubkey is encode to 34 bytes which startd with OPCODE and OP_PUSHBYTES_32 these two bytes

**Comment by dr-orlovsky** [comment]
From what I understand it is the same issue which is fixed in https://github.com/RGB-WG/rgb-std/pull/210

**Comment by dr-orlovsky** [comment]
Well, we do not encode the whole address, we just encode the schnorr key - the address payload. Thus, this is a no-issue (unless something is not working for you)

**Comment by dr-orlovsky** [comment]
Closed in #210
